### PR TITLE
Change IDREF to IDREFS

### PR DESCRIPTION
**Pull request by jdmgoogle**
Change IDREF to IDREFS for all elements which are both IDREF and maxOccurs=unbounded.

This addresses issue #298.

Paging @cjerdonek to make sure I generated the RST from the YAML correctly.

### CODE REVIEW

File: docs/yaml/elements/locality.yaml [hunk 1]
```diff
@@ -16,15 +16,14 @@ tags:
     ignore the Locality element containing it.
   required: true
   type: xs:string
-- _name: PollingLocationId
-  description: Specifies a link to the locality's :doc:`polling locations <polling_location>`.
-    If early vote centers or ballot drop locations are locality-wide, they should
-    be specified here.
+- _name: PollingLocationIds
+  description: Specifies a link to a set of the locality's :doc:`polling locations
+    <polling_location>`s. If early vote centers or ballot drop locations are locality-wide,
```
Comment (jungshadow) [inline docs/yaml/elements/locality.yaml]: Looks like this might be a typo. Did you mean:
```
:doc:`polling location <polling_location>`(s)
```

**Comment by jungshadow** [comment]
Besides the one typo, LGTM (not that my opinion matters much these days) :)

**Comment by cjerdonek** [comment]
> Paging @cjerdonek to make sure I generated the RST from the YAML correctly.

Looks okay!

**Comment by jdmgoogle** [comment]
Thanks. I'm going to hold off merging this until after #301.

Paging @pstenbjorn.

**Comment by pstenbjorn** [comment]
@jdmgoogle LGTM, looking at #300 now

**Comment by jdmgoogle** [comment]
Okay @pstenbjorn and @cjerdonek, this appears to be ready for review.

**Comment by pstenbjorn** [comment]
@jdmgoogle LGTM.

File: docs/yaml/elements/state.yaml [hunk 2]
```diff
@@ -33,11 +33,10 @@ tags:
   error_then: =must-ignore
   required: true
   type: xs:string
-- _name: PollingLocationId
-  description: Specifies a link to the state's :ref:`polling locations <$$$-polling-location>`.
+- _name: PollingLocationIds
+  description: Specifies a link to the state's :ref:`polling locations <$$$-polling_location>`.
```
Comment (cjerdonek) [inline docs/yaml/elements/state.yaml]: This should be a hyphen. O/w looks good to me!

File: docs/yaml/elements/ordered_contest.yaml [hunk 3]
```diff
@@ -21,11 +21,10 @@ tags:
     ignore the ``OrderedContest`` element containing it.
   required: true
   type: xs:IDREF
-- _name: OrderedBallotSelectionId
+- _name: OrderedBallotSelectionIds
   description: Links to elements that extend :ref:`$$$-ballot-selection-base`.
   error: If the field is invalid or not present, the implementation is required to
-    ignore it. If no ``OrderedBallotSelectionId`` elements are present, the presumed
+    ignore it. If a ``OrderedBallotSelectionIds`` element is not present, the presumed
```
Comment (cjerdonek) [inline docs/yaml/elements/ordered_contest.yaml]: Also, "an" OrderedBallot...

**Comment by jdmgoogle** [comment]
Unless @cjerdonek has any particular complaints with this, I plan to merge this PR by noon PT.

**Comment by cjerdonek** [comment]
Looks good!

**Pull request merged**